### DISCUSSION TITLE
Remove support for old Windows release

### DISCUSSION THREAD

**Discussion by ansani** [ansani/Shareaza#23] [Polls]
Do you think I should drop support for old releases of Windows?
I think I can continue support from Windows 8.1 so we can easily add new features to the client.

**Comment by ivan386** [comment]
Can't vote because GitHub drop support of firefox 68. I for support older release. And dont forget that Shareaza must work under Wine for Linux.

**Reply by ansani** [reply]
Hi Ivan! Thank you for your effort! I basically started with your changes. About the older release, I still don't have a plan because I'm exploring different paths. About the Wine support, it's still a goal and I will add a specific discussion/section to handle issues.
